src/toil/jobStores/aws/jobStore.py

```diff
@@ -1418,7 +1422,10 @@ def download(self, localFilePath, verifyChecksum=True):
                 assert False
 
         @contextmanager
-        def downloadStream(self, verifyChecksum=True):
+        def downloadStream(self, verifyChecksum=True, encoding=None, errors=None):
+            """
+            Context manager that gives out a binary/text mode download stream to download data.
```

Comment:
I guess you could just say `a download stream`, since binary and text are two opposite modes a stream can have.